docs/pages/_app.js

```diff
@@ -29,6 +29,7 @@ import findActivePage from 'docs/src/modules/utils/findActivePage';
 import { pathnameToLanguage } from 'docs/src/modules/utils/helpers';
 import getProductInfoFromUrl from 'docs/src/modules/utils/getProductInfoFromUrl';
 import './global.css';
+import './base-theme.css';
```

Comment:
https://nextjs.org/docs/app/building-your-application/styling/css-modules

<img src="https://github.com/mui/material-ui/assets/3165635/6875496d-0ea5-4f54-ac74-c64c7a89ba9e" width="658">

```suggestion
// TODO app directory migration: move import to relevant page
import './base-theme.css';
```